assets/js/components/ClusterDetails/ClusterDetailsNew.jsx

```diff
@@ -0,0 +1,272 @@
+            onClick={() => navigate(`/clusters/${clusterID}/checks/results`)}
+          >
+            <EOS_CLEAR_ALL className="inline-block fill-jungle-green-500" />{' '}
+            Show Results
+          </Button>
+          <TriggerChecksExecutionRequest
+            cssClasses="rounded relative w-1/4 ml-0.5 disabled:bg-slate-50 disabled:text-slate-500 disabled:border-gray-400"
+            clusterId={clusterID}
+            disabled={!hasSelectedChecks}
+          >
+            <EOS_PLAY_CIRCLE
+              className={classNames('inline-block fill-jungle-green-500', {
+                'fill-slate-500': !hasSelectedChecks,
+              })}
+            />{' '}
+            Start Execution
+            {!hasSelectedChecks && (
+              <Tooltip tooltipText="Select some Checks first!" />
+            )}
+          </TriggerChecksExecutionRequest>
+        </div>
+      </div>
+
+      <div className="flex xl:flex-row flex-col">
+        <div className="tn-cluster-details mt-4 bg-white shadow rounded-lg py-8 px-8 xl:w-3/4 w-full mr-4">
+          <ListView
+            className="grid-rows-3"
+            titleClassName="text-lg"
+            orientation="vertical"
+            data={[
+              {
+                title: 'Provider',
+                content: cluster.provider || 'Not defined',
+                render: (content) => <ProviderLabel provider={content} />,
+              },
+              { title: 'SID', content: cluster.sid },
+              {
+                title: 'Fencing type',
+                content: cluster.details && cluster.details.fencing_type,
+              },
+              {
+                title: 'Cluster type',
+                content:
+                  cluster.type === 'hana_scale_up'
+                    ? 'HANA scale-up'
+                    : 'Unknown',
+              },
+              {
+                title: 'SAPHanaSR health state',
+                content: cluster.details && cluster.details.sr_health_state,
+              },
+              {
+                title: 'CIB last written',
+                content: cluster.cib_last_written || '-',
+              },
+              {
+                title: 'HANA log replication mode',
+                content:
+                  cluster.details && cluster.details.system_replication_mode,
+              },
+              {
+                title: 'HANA secondary sync state',
+                content:
+                  cluster.details && cluster.details.secondary_sync_state,
+              },
+              {
+                title: 'HANA log operation mode',
+                content:
+                  cluster.details &&
+                  cluster.details.system_replication_operation_mode,
+              },
+            ]}
+          />
+        </div>
+        <div className="tn-cluster-checks-overview mt-4 bg-white shadow rounded-lg py-4 xl:w-1/4 w-full">
+          <ChecksResultOverviewNew
+            {...lastExecution}
```

Comment:
Yikes!

Reply:
(given that `explicit is better than implicit` your call if you want to just splat this or set prop by prop)

Reply:
> `explicit is better than implicit`

Follow me for more proverbs